lib/survey.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+while true; do
+
+echo "Do you want to proceed? (yes/no) "
```

Comment:
Perhaps it would be more correct to clarify: `Are you ready to be a Dalek?`

I used 'to proceed' for the purpose of universal use in the future. Maybe use a simple: ~~get ready to the next battle~~ `Continue?`

Reply:
I think if you plan to use this script in the future - you can add parameter to the script 'promot' and inside the restore script - you can send the promt parameter 'This will *delete current DB* and restored backup. Are you ready?'

Reply:
Let's leave it like now - and if we need to use this promt again - extract to separate file